apiary.apib

```diff
@@ -3617,6 +3617,7 @@ Get details for a single work order.
                     + id: `a856642a-a476-4b69-8e42-e7d1badb2ce9` (string)
                     + location: `https://cdn.teamleader.eu/file` (string) - A temporary URL where the image can be downloaded.
                     + expires: `2019-05-09T13:12:42+00:00` (string) - Expiration time of the temporary download URL.
+            + signed: false (boolean)
```

Comment:
PP: place this one just before the meta properties `created_at` and `updated_at`.